go-controller/pkg/ovn/egressip.go

```diff
@@ -1600,6 +1600,14 @@ func (oc *Controller) setNodeEgressReachable(nodeName string, isReachable bool)
 
 func (oc *Controller) addEgressNode(nodeName string) error {
 	var errors []error
+	// Check if EgressIP node create failed and if does try adding it again
+	if node, ok := oc.addEgressIPNodeFailed.Load(nodeName); ok {
+		failedNode := node.(*kapi.Node)
+		if err := oc.addNodeForEgress(failedNode); err != nil {
+			return err
+		}
+		oc.addEgressIPNodeFailed.Delete(nodeName)
```

Comment:
Since we are already doing `oc.addEgressIPNodeFailed.Delete(node.Name)` in line 1754, isn't this a bit redundant?

Reply:
you are right no need for this additional delete will remove